packages/platform/types/event-types/event-types_2024_06_14/inputs/requires-confirmation.input.ts

```diff
@@ -0,0 +1,92 @@
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { IsEnum, IsOptional, IsInt, ValidateNested, IsBoolean } from "class-validator";
+import type { ValidatorConstraintInterface, ValidationOptions } from "class-validator";
+import { ValidatorConstraint, registerDecorator } from "class-validator";
+
+import { ConfirmationPolicyEnum, NoticeThresholdUnitEnum } from "@calcom/platform-enums";
+
+// Class representing the notice threshold
+export class NoticeThreshold_2024_06_14 {
+  @IsInt()
+  @ApiProperty({
+    description: "The time value for the notice threshold",
+    example: 30,
+  })
+  time!: number;
```

Comment:
For clarity, i would put `unit` above `time` and rename `time` to `count` or `amount`.

Reply:
fixed https://github.com/calcom/cal.com/commit/305de6fc68d1e1c9ea62badf3e5ae88e5fc3ec60